explorer/explorer.go

```diff
@@ -684,3 +685,14 @@ func (exp *explorerUI) simulateASR(StartingDCRBalance float64, IntegerTicketQty
 	ReturnTable += fmt.Sprintf("ASR over 365 Days is %.2f.\n", ASR)
 	return
 }
+
+// genesisBlockTime returns the genesis block time.
+func (exp *explorerUI) genesisBlockTime() (int64, error) {
+	hash, err := exp.blockData.GetBlockHash(0)
+	if err != nil {
+		return -1, err
+	}
+
+	data := exp.blockData.GetExplorerBlock(hash)
```

Comment:
Also might as well skip the `GetBlockHash(0)` and use `params.GenesisHash`.  Even better, use `params.GenesisBlock.Header.Timestamp` and avoid any DB look ups or issues getting the value before genesis is in the db.